R/cnvrt-methods.R

```diff
@@ -50,7 +50,16 @@ setAs(from="phylo", to="TreeMan", def=function(from, to) {
 #' tree <- as(tree, 'TreeMan')
 setAs(from="TreeMan", to="phylo", def=function(from, to) {
   temp_file <- tempfile(pattern="temp_tree", fileext=".tre")
-  writeTree(from, file=temp_file)
+  
+  # use a node label function function that writes the node labels during the 
+  # conversion, if they are non default.
+  if (all(from["nds"] == paste0("n", seq_along(from["nds"])))) {
+    node_labeller <- function(nd) { return(NULL) } # no node labels 
+  } else {
+    node_labeller <- function(n) { print(n); n[['id']] } # writes node labels 
```

Comment:
```suggestion
    nodeLabeller <- function(n) { n[['id']] } # writes node labels 
```
I don't thinking printing is necessary.

Reply:
Definitely not ...